services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/BasePreservedKeyChecker.java

```diff
@@ -16,6 +21,16 @@ public boolean isKeyPreserved(String key) {
     return IS_OH_PREFIXED.test(key);
   }
 
+  /**
+   * Details of determining if a key is preserved for the given {@link TableDto} is hidden in the
```

Comment:
can you also explain whats the behavior for `@TblPropsEnabler` if its disabled? will it throw exception/return true etc in this doc?

Reply:
Adding this annotation will just have an interceptor examine the result and examine the result together whether there's a feature being active. Here the feature is allowing some keys to be modified. Not sure if that answers the question.